tasks/ast_codegen/src/generators/assert_layouts.rs

```diff
@@ -46,6 +46,10 @@ impl Generator for AssertLayouts {
             quote! {
                 #header
 
+                use std::mem::{align_of, offset_of, size_of};
```

Comment:
`align_of` and `size_of` are added to prelude from `1.80.0`

Reply:
See below...